test-harness/scenarios/results-skip-rule.oas3.scenario

```diff
@@ -15,4 +15,4 @@ OpenAPI 3.x detected
  1:1  warning  api-servers       OpenAPI `servers` must be present and non-empty array.
  2:6  warning  info-description  OpenAPI object info `description` must be present and non-empty string.
 
-✖ 2 problems (0 errors, 2 warnings, 0 infos)
\ No newline at end of file
+✖ 2 problems (0 errors, 2 warnings, 0 infos, 0 hints)
```

Comment:
Should we just tackle proper pluralization for the values in this message in this PR? Suggest either:

- https://github.com/prantlf/fast-plural-rules
- https://github.com/blakeembrey/pluralize

Reply:
We already have a simple util for that.
```ts
function pluralize(word: string, count: number): string {
  return count === 1 ? word : `${word}s`;
}
```
I guess this should be quite enough for the simple case we deal with?

Reply:
Yeah that sounds good to me!